elide-core/src/main/java/com/yahoo/elide/core/PersistentResource.java

```diff
@@ -509,6 +508,12 @@ protected boolean updateToManyRelation(String fieldName,
                 });
 
 
+        //Hook for setRelation
+        Class<?> targetClass = obj.getClass();
+        if (dictionary.isRelation(targetClass, fieldName)) {
+            transaction.setRelation(transaction, obj, fieldName, collection, requestScope);
+        }
+
```

Comment:
I'm not sure if the setrelation hook should also go into `addInverseRelation()`, since it's also updating the inverse relation.